hbase-server/src/main/java/org/apache/hadoop/hbase/io/hfile/PrefetchProtoUtils.java

```diff
@@ -26,8 +28,26 @@ private PrefetchProtoUtils() {
   }
 
   static PersistentPrefetchProtos.PrefetchedHfileName
-    toPB(Map<String, Boolean> prefetchedHfileNames) {
-    return PersistentPrefetchProtos.PrefetchedHfileName.newBuilder()
-      .putAllPrefetchedFiles(prefetchedHfileNames).build();
+    toPB(Map<String, Pair<String, Long>> prefetchedHfileNames) {
```

Comment:
It is hard for me to find out what is in `Map<String, Pair<String, Long>>`. Is it possible to replaces at least `Pair<String, Long>` with a class?